analyses/cms-open-data-ttbar/coffea.ipynb

```diff
@@ -7,6 +7,8 @@
    "source": [
     "# CMS Open Data $t\\bar{t}$: from data delivery to statistical inference\n",
     "\n",
+    "<span style=\"color:red\"> *NOTE: this is an old version of the CMS ttbar analysis notebook. Development on this notebook has moved to `analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.ipynb`.* </span>\n",
```

Comment:
Could we please remove anything below this in the notebook to not keep duplicated code around?

Reply:
Done!